hooks/post_gen_project.py

```diff
@@ -1,9 +1,149 @@
+import os
+import json
+import shutil
 import subprocess
 
+from python_terraform import Terraform
+
+YES_OPTIONS, NO_OPTIONS = frozenset(["y"]), frozenset(["n", ""])
+PERM_OPTIONS = frozenset(["push", "pull", "admin"])
+
+
+def multiple_choices(question):
+    selection = []
+    additional = "Would you like to add another (y/n)? [n]: "
+
+    multi_q_check = True
+    while multi_q_check:
+        choice = input(question).lower()
+        selection.append(choice)
+        multi_q_check = check_choice(additional)
+    return selection
+
+
+def check_choice(question):
+    choice = input(question).lower()
+    if choice in YES_OPTIONS:
+        return True
+    if choice in NO_OPTIONS:
+        return False
+
+
+def get_collaborators():
+    # Questions that have multiple layers to them
+    collaborators = []
+    collab_q = "Grant an organizational team access to the repo(y/n)? [n]: "
+
+    if check_choice(collab_q):
+        add_team = True
+
+        while add_team:
+            teams_q = "What team would you like to grant access? "
+            team = input(teams_q).lower()
+
+            if team != "":
+                perm = None
+                while perm not in PERM_OPTIONS:
+                    perms_q = ("What permissions would you like to grant them (push, pull, admin)? ")
+                    perm = input(perms_q).lower()
+                collaborators.append({"name": team, "permission": perm})
+
+            another_q = "Would you like to add another team (y/n)? [n]: "
+            add_team = check_choice(another_q)
+
+    return collaborators
+
+
+def run_terraform(directory):
+    terraform = Terraform(directory)
+    terraform.init()
+    ret_code, _, _ = terraform.apply(auto_approve=True, capture_output=False)
+
+    return ret_code
+
+
+def remove_terraform_files(directory):
+    file_names = [
```

Comment:
yeah, don't space these over so much...

```suggestion
    file_names = [
        "main.tf",
        "variables.tf",
        "terraform.tfvars",
        "terraform.tfstate"
    ]
```